propagation_ot.go

```diff
@@ -52,9 +52,11 @@ func (p *textMapPropagator) Inject(
 	carrier.Set(zipkinTraceID, strconv.FormatUint(sc.raw.TraceID, 16))
 	carrier.Set(zipkinSpanID, strconv.FormatUint(sc.raw.SpanID, 16))
 	if !p.tracer.options.clientServerSameSpan {
-		sc.raw.ParentSpanID = 0
+		carrier.Set(zipkinParentSpanID, "0")
```

Comment:
Hi, @adriancole. Have you read my issue and correctly understood the problem I`ve found? If I pass span with:
```
traceID: 123
parentID: 456
spanID: 789
```

After this call it becomes:
```
traceID: 123
parentID: 0
spanID: 789
```

And it zipkin I`ll see this span without any parent.

> In http B3 propagation, we can represent null (absence of the parent header). By serializing the value "0", which isn't null, some tracers may be fooled into thinking the span has a parent. Remember these ids are opaque bytes serialized into 64bits of hex. There's nothing special about "0"

ParentSpanID cannot be null because it is just `uint64` not `*uint64`. Before my changes this code did 2 things:
1. Set parentSpanID to 0 despite its value
2. convert this "0" to string and pass it to carrier. (https://play.golang.org/p/QMJ8JOCVcI)

So I`ve just removed unnecessary calls and conversions. If in this case it`s better not to pass ParentSpanID ok, I can do it.